scout/commands/load/evaluation_terms.py

```diff
@@ -0,0 +1,50 @@
+"""CLI for adding new evalution terms to the database."""
+
+import logging
+
+import click
+from flask.cli import with_appcontext
+
+from scout.load.evaluation_terms import (
+    load_default_evaluation_terms,
+    load_evaluation_terms_from_file,
+)
+from scout.server.extensions import store
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group()
+def evaluation_terms():
+    """Update custom or default evaluation terms"""
+
+
+@evaluation_terms.command()
+@with_appcontext
+def default_terms():
+    """Load default variant evaluation terms into database"""
+
+    click.confirm(
+        'This command will replace all evantual variant evaluation terms ("dismissal_term", "manual_rank", "cancer_tier", "mosaicism_options") present in the database with the default ones. Continue?',
+        abort=True,
+    )
+    load_default_evaluation_terms(store)
+
+
+@evaluation_terms.command()
+@with_appcontext
+@click.option(
+    "-f",
+    "--file",
+    type=click.File(),
+    required=True,
+    help="Load a python file with multiple evaluation terms",
+)
+def custom_terms(file):
+    """Load custom variant evaluation terms from a json file into database"""
+
+    click.confirm(
+        'This command will replace all evantual variant evaluation terms ("dismissal_term", "manual_rank", "cancer_tier", "mosaicism_options") present in the database with the ones found on the provided file. Continue?',
+        abort=True,
+    )
+    load_evaluation_terms_from_file(store, file)
```

Comment:
Consider catching user errors like duplicated entry or invalid data format and print them nicely, without a stacktrace. You could add custom exception classes for that.

Reply:
I think it's fine, it's not that we are going to change those keys every week. We in Stockholm are not even probably touching them again. If we change one key and it's not valid or it is duplicated the error is triggered already.

Reply:
I have a feeling we will touch them again - at least the longer descriptions - but we should be able to manage without much error handling. It is config that still leans fairly heavily on the system/dev side.

Reply:
If you try to load a duplicated key for the same category or an invalid term you get ValueError already